3-semantic-html/basic.html

```diff
@@ -0,0 +1,46 @@
+<html lang="en">
```

Comment:
```suggestion
<!DOCTYPE html>
<html lang="en">
```
Doctype is required for browsers compatibility and document validation.